src/topp/OpenSwathFeatureXMLToTSV.cpp

```diff
@@ -279,11 +292,16 @@ void write_out_body_(std::ostream &os, Feature *feature_it, TargetedExperiment &
     char mz_char[40];
     for (std::vector<Feature>::iterator sub_it = feature_it->getSubordinates().begin(); sub_it != feature_it->getSubordinates().end(); ++sub_it)
     {
-      String Peak_Apex = "NA";
       os.precision(writtenDigits(double()));
       sprintf(intensity_char, "%f", sub_it->getIntensity());
       sprintf(mz_char, "%f", sub_it->getMZ());
-      os << line << meta_values << (String)intensity_char << "\t" << Peak_Apex << "\t" << (String)sub_it->getMetaValue("native_id") << "\t" << (String)mz_char << std::endl;
+      String apex = "NA";
+      if (sub_it->metaValueExists("peak_apex_int"))
+      {
+        sprintf(intensity_apex_char, "%f", (double)sub_it->getMetaValue("peak_apex_int"));
+        apex = (String) intensity_apex_char;
+      }
+      os << line << meta_values << (String)intensity_char << "\t" << apex << "\t" << (String)sub_it->getMetaValue("native_id") << "\t" << (String)mz_char << std::endl;
```

Comment:
Just out of curiosity, in OpenSWATH, do you connect spectra via a string "native_id" or is it a numerical native id or an index to the file?